pkg/graph/graph.go

```diff
@@ -239,7 +243,7 @@ func handleIncomingEdges[V Identifiable, O any](d *Directed[V], to V, generate f
 	for _, v := range fullAdjacency {
 		for _, edge := range v {
 			if edge.Target != to.Id() {
-				zap.S().Debugf(`Ignoring unexpected edge source from %v`, edge)
+				// zap.S().Debugf(`Ignoring unexpected edge target from %v`, edge)
```

Comment:
intentionally commented out? if so, just rm it.

Reply:
nope, let me add it back, was just endlessly printing because of circular dep. Going to add to this pr though as well